### ISSUE TITLE
Storing notifications during quiet time

### ISSUE THREAD

**Issue by Telhop** [pilot51/voicenotify#109]
Firstly thanks for a great app. I have been using it since you first developed it. I wondered if you might consider adding an option to allow notifications to be stored during the 'quiet time' and read out after VN is scheduled to become active again.

**Comment by pilot51** [comment]
I like this idea. It was mentioned in #103 where I suggested creating a separate issue for it, which this happens to be, so thank you.

The storage portion of this will most certainly be shared with #95.

Not sure when I'll get around to it with my current plans, I want to get a couple big structural changes done before focusing on new features, so don't expect it very soon.

**Comment by rakolambski** [comment]
Since the notifications are "still there" (until the user handles/deletes them), another option might be to just read the then current notifications once quiet time is over. 

Triggering a *"speak all notifications as if new"* feature (if selected via an option) once quiet time ends could accomplish this, *assuming there's a way to see a queue of current notifications.*

Actually "storing" notifications might cause confusion, if you later read a notification that the user has already dealt with (and which would have been deleted from the notification list.)

**Comment by Telhop** [comment]
I would agree that reading the current notifications at the end of quiet time makes more sense than storing them.
